tools/desktop-scripts/measure_word.pl

```diff
@@ -1,100 +0,0 @@
-# **********************************************************
```

Comment:
> i#5369: Delete obsolete files, part 1

Since this is deleting one specific file, IMHO that should be in the title.  Also note that the "part 1" phrase is normally used in the title prefix before the colon.

Reply:
Done.